docs/ref/validators.txt

```diff
@@ -145,6 +145,23 @@ to, or in lieu of custom ``field.clean()`` methods.
         ``['localhost']``. Other domains that don't contain a dot won't pass
         validation, so you'd need to whitelist them as necessary.
 
+    The :class:`EmailValidator` works in accordance with the `HTML5 email
+    validation standard`_.
+
+    .. _HTML5 email validation standard: https://html.spec.whatwg.org/multipage/forms.html#valid-e-mail-address
+
+    .. versionchanged:: 2.0
+
+        In older version, email validation was done with Django's own
+        logic. Since version 2.0, it uses the HTML5 email validation
+        standard.
+
+	The ``whitelist`` attribute is deprecated since 2.0. It was
```

Comment:
Should be documented under `.. attribute:: whitelist` -- although it's removed so maybe it should be deleted instead.